rules-tests/TypeDeclaration/Rector/ClassMethod/AddParamTypeFromPropertyTypeRector/Fixture/variadic_constructor_param_of_explicit_type.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddParamTypeFromPropertyTypeRector\Fixture;
+
+final class SkipVariadicConstructorParamOfExplicitType
+{
+    /**
+     * @var DateTime[]
+     */
+    private array $dates;
+
+    public function __construct(...$dates) {
+        $this->dates = $dates;
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddParamTypeFromPropertyTypeRector\Fixture;
+
+final class SkipVariadicConstructorParamOfExplicitType
+{
+    /**
+     * @var DateTime[]
+     */
+    private array $dates;
+
+    public function __construct(...$dates) {
```

Comment:
@samsonasik 

Instead of 

`public function __construct(...$dates) {`

`public function __construct(DateTime ...$dates) {`

would be nice. But I couldn't find a way how to read from the dock block comment. 

Any idea?

Reply:
we can't rely on docblock, so skipping is the safest.